lib/gpio/adc.toit

```diff
@@ -42,12 +44,18 @@ class Adc:
   Note that chip-specific limitations apply, generally the precision at
     various voltage ranges.
 
-  On the ESP32, only the ADC1 (pins 32-39) is supported. The ADC2 has too many
-    restrictions (cannot be used when WiFi is active, and some of the pins are
-    strapping pins), and is therefore disabled.
+  If $allow_dangerous is provided, actives ADC pins that are dangerous to use.
```

Comment:
Is it really dangerous or just unreliable, unstable, restricted, special-purpose?